### ISSUE TITLE
Unable to print labels at all

### ISSUE THREAD

**Issue by rm3dom** [dymosoftware/DCD-SDK-Sample#68]
Please see sample project here:

https://github.com/rm3dom/dymo_connect_fails

Tested on Windows 10

The README.md describes the issue.

**Comment by bubblesnout** [comment]
You need to run the following to initialize the SDK before you can use it properly:

`DymoSDK.App.Init();`

**Comment by rm3dom** [comment]
> You need to run the following to initialize the SDK before you can use it properly:
> 
> `DymoSDK.App.Init();`

Fantastic, thank you, will give it a go...

**Comment by bubblesnout** [comment]
I just put it in my program startup function, appears that it only needs to be run once.

**Comment by rm3dom** [comment]
Yeah haven't tried it, but it does sound like what I was missing. They load assemblies at runtime and cache classes in a dict. The dict was empty in my case
